examples/jsm/node-editor/materials/BasicMaterialEditor.js

```diff
@@ -1,14 +1,14 @@
 import { ColorInput, SliderInput, LabelElement } from '../../libs/flow.module.js';
-import { BaseNode } from '../core/BaseNode.js';
+import { MaterialEditor } from './MaterialEditor.js';
 import { MeshBasicNodeMaterial } from 'three/nodes';
 
-export class BasicMaterialEditor extends BaseNode {
+export class BasicMaterialEditor extends MaterialEditor {
 
 	constructor() {
 
 		const material = new MeshBasicNodeMaterial();
 
-		super( 'Basic Material', 1, material );
+		super( 'Basic Material', material );
 
 		this.setWidth( 300 );
```

Comment:
Maybe we can remove this and similar lines in other materials editors if MaterialEditor already sets width to 300?